### DISCUSSION TITLE
能不能开发一个手机版，支持调节速度等基本功能？

### DISCUSSION THREAD

**Discussion by Iamyourbaba** [the1812/Bilibili-Evolved#3306] [功能建议 / Ideas]
能不能开发一个手机版，满足一下最基本的功能，比如，调节速度这个最基础的就行

**Comment by the1812** [comment]
不能

**Comment by Geeyun-JY3** [comment]
手机网页端这种残废你真的用得下去吗……

而且我看有些支持油猴的浏览器也有调播放速度的通用脚本

**Comment by hmgdd101** [comment]
手机端有的用，via的脚本功能立刻，开电脑模式

**Comment by CaseYwY** [comment]
所以为什么不直接用b站的app？

**Comment by theblueandblackcrow** [comment]
> 能不能开发一个手机版，满足一下最基本的功能，比如，调节速度这个最基础的就行

b站app：你找我？
